script/Arcueid/Day 13/13_00_ARC13_1.txt

```diff
@@ -285,7 +285,7 @@ Akiha jumps straight to the heart of the matter.
 [sha:31032461f2bea04259a851e30de154d2a9a7eed2]{
 -- Page 18, Offset 31723.
 -- ……もっとも、聞かなくとも兄さんの容態を見れば一目でわかる事ですが」
-...Though, even if I hadn't, it'd be clear from one look at you what had transpired."
+...Though, even if I didn't, it'd be clear from one look at you what had transpired."
```

Comment:
revert

Reply:
:/ previous line says "heard", wouldn't I use the same tense if I were to negate it?

Reply:
I agree with hadn't, because the implication is
> Even if I hadn't [heard]

and 
> Even if I didn't heard

is no good